packages/plugin-brotli/package.json

```diff
@@ -1,15 +1,14 @@
 {
   "name": "@pwa/plugin-brotli",
-  "version": "0.5.0",
+  "version": "0.5.1",
   "files": [
     "index.js"
   ],
   "engines": {
-    "node": ">=8"
+    "node": ">= 10.13.0"
```

Comment:
This is a breaking change. 

Is there any way to get this working with [`compression-webpack-plugin@4.x`](https://github.com/webpack-contrib/compression-webpack-plugin/releases/tag/v4.0.0) since that was the last version that's compatible with Node 8.x?

If not, we'll have to hold onto this PR and update _everything_ at once so that it's all sharing the same Node and webpack minimum requirements.

Reply:
I'll wait then. I prefer to stick with Node's latest LTS when possible.